ports/libavif/portfile.cmake

```
@@ -1,15 +1,16 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO AOMediaCodec/libavif
-    REF v0.9.2
-    SHA512 04400ae76214d2f0361a14897d6ee97be675375865bb96c8d237e9a4a1152ac1a966db903c11df82da71b0bc68599a5857e038cc90d63c5d3bc77b13169a3e75
+    REF f938f6b7dd412ffcbdab1808a265b6375dc7b037 #v0.10.1
+    SHA512 d4e01edb9891df0b0abc4a1d621287bce6ba38248a7ae458abd73c268b000557a52a5aa6be1fc715b694c8e48166aee458a371d806d5f28160c50ff653420e79
     HEAD_REF master
     PATCHES
         disable-source-utf8.patch
+        fix-compiler-warnings.patch

```

```suggestion
        fix-compiler-warnings.patch #Remove this patch in next update
```
The patch from upstream PR: https://github.com/AOMediaCodec/libavif/pull/660